saleor/graphql/product/mutations/products.py

```diff
@@ -341,11 +358,21 @@ def user_is_allowed(cls, user, input):
 
 
 class ProductTypeInput(graphene.InputObjectType):
-    name = graphene.String()
-    has_variants = graphene.Boolean(required=True)
-    product_attributes = graphene.List(graphene.ID)
-    variant_attributes = graphene.List(graphene.ID)
-    is_shipping_required = graphene.Boolean(required=True)
+    name = graphene.String(description='Name.')
+    has_variants = graphene.Boolean(
+        required=True,
+        description="""Determines if variants are allowed
```

Comment:
We may provide some more explanation here: "Determines if product of this type has multiple variants. This option mainly simplifies product management in the dashboard. There is always at least one variant created under the hood."